swagger/body-time.json

```diff
@@ -0,0 +1,86 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "AutoRest Time Test Service",
+      "description": "Test Infrastructure for AutoRest",
+      "version": "1.0.0"
+    },
+    "host": "localhost:3000",
+    "schemes": [
+      "http"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "consumes": [
+      "application/json"
+    ],
+    "paths": {
+      "/time/get": {
+        "get": {
+          "operationId": "time_get",
+          "description": "Get time value",
+          "responses": {
+            "200": {
+              "description": "The time value",
+              "schema": {
+                "type": "string",
+                "format": "time"
+              }
+            },
+            "default": {
+              "description": "Unexpected error",
+              "schema": {
+                "$ref": "#/definitions/Error"
+              }
+            }
+          }
+        }
+      },
+      "/time/put": {
+        "put": {
+          "operationId": "time_put",
+          "description": "Put time value",
+          "parameters": [
+            {
+              "name": "timeBody",
```

Comment:
Can you also include the required parameter value in description.

Reply:
It's in the description of the operation, you want it in the description of the parrameter too?